app/views/SMA-prototype/sma.html

```diff
@@ -29,7 +29,10 @@
 
     <div class="govuk-grid-row">
       <div class="govuk-grid-column-full">
-        <h1 class="govuk-heading-xl">Spinal Muscular Atrophy (SMA) <span id="type-tag">Adult</span></h1>
+        <h1 class="govuk-heading-xl">
+            <span class="govuk-caption-xl">Adult screening programme</span>
```

Comment:
The current code has this caption/tag element inside the `<h1>`, but I'm just wondering if it should be outside. Not sure if it's part of the title or not.

Could probably be argued either way!